### PR TITLE
#64 | Admin assign validators

### PR DESCRIPTION
**Pull request by khlyy**
-made new Admin Controller .
-made new routing for the admin .
-removing unUsed lines.

### CODE REVIEW

File: resources/views/volunteer/show.blade.php [hunk 1]
```diff
@@ -61,5 +61,9 @@
                     <a href="{{ action('VolunteerController@viewReviews', [$volunteer->id])}}">View More >></a>
             @endif
         </ul>
-    </ul>
```
Comment (AhmadElsagheer) [inline resources/views/volunteer/show.blade.php:64]: restore this line

File: app/Http/routes.php [hunk 2]
```diff
@@ -180,11 +180,11 @@
         'show', 'edit', 'update'
     ]]);
 
-/*
-|-----------------------
-| Event Routes
-|-----------------------
-*/
```
Comment (AhmadElsagheer) [inline app/Http/routes.php:187]: restore these lines

File: app/Http/routes.php [hunk 3]
```diff
@@ -226,4 +226,15 @@
      *  Event CRUD.
      */
     Route::resource('event','EventController', ['except' => 'index']);
+
+    /*
+    |-----------------------
+    | Admin Routes
+    |-----------------------
+    */
```
Comment (AhmadElsagheer) [inline app/Http/routes.php]: fix identation to match the rest the documentation of this file

File: app/Http/Controllers/AdminController.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (AhmadElsagheer) [inline app/Http/Controllers/AdminController.php]: remove all these extra lines

**Comment by AhmadElsagheer** [comment]
One file needs renaming: views/volunteer/adminPanel/Assign Validator.blade.php => views/admin/assign-validator.blade.php

NOTE: don't close this pull request. Just push edits on branch

**Comment by khlyy** [comment]
5las l7l :dancer:

**Pull request merged**